### DISCUSSION TITLE
`_interopRequireDefault` and `_interopRequireWildcard`  should be marked as pure

### DISCUSSION THREAD

**Discussion by magic-akari** [swc-project/swc#5051] [General]
```javascript
const _foo = /*#__PURE__*/_interopRequireDefault(require("foo"));
const _bar = /*#__PURE__*/_interopRequireWildcard(require("bar"));
```

There are two methods to do that.

1. add comments param to CJS/AMD/UMD parmas, mark them as they are constructed.
2. add a new pass to find the two callee.

I'm not sure which is better.

**Comment by kdy1** [comment]
I think 1 is logically better, but it would require more work
